libsodium-sys/src/crypto_aead_chacha20poly1305.rs

```diff
@@ -24,13 +24,34 @@ extern {
     pub fn crypto_aead_chacha20poly1305_decrypt(
         m: *mut u8,
         mlen: *mut c_ulonglong,
-        nsec: *mut [u8; crypto_aead_chacha20poly1305_NSECBYTES],
+        nsec: *const [u8; crypto_aead_chacha20poly1305_NSECBYTES],
         c: *const u8,
         clen: c_ulonglong,
         ad: *const u8,
         adlen: c_ulonglong,
         npub: *const [u8; crypto_aead_chacha20poly1305_NPUBBYTES],
         k: *const [u8; crypto_aead_chacha20poly1305_KEYBYTES]) -> c_int;
+    pub fn crypto_aead_chacha20poly1305_encrypt_detached(
+        c: *mut u8,
+        mac: *mut u8,
+        maclen: *mut c_ulonglong,
+        m: *const u8,
+        mlen: c_ulonglong,
+        ad: *const u8,
+        adlen: c_ulonglong,
+        nsec: *const [u8; crypto_aead_aes256gcm_NSECBYTES],
```

Comment:
libsodium doesn't define `nsec` as const. I agree that it should be, but we need to follow the actual API so we don't get any surprises if libsodium decides to start mutating `nsec`.